nixos/modules/services/web-servers/nginx/location-options.nix

```diff
@@ -7,6 +7,7 @@
 
 with lib;
 
+let self =
```

Comment:
Why this?

Reply:
You are right, I forgot to actually use `self`. Fixed that.